data-modeling/delegate-types/README.md

```diff
@@ -0,0 +1,68 @@
+# Data modeling: Delegate types
+
+This example shows you how to model [delegate types](https://edgeapi.rubyonrails.org/classes/ActiveRecord/DelegatedType.html) using Prisma, and use [Prisma Client](https://www.prisma.io/client) in a **TypeScript script** to read and write data.
+
+This example consists of a `script.ts` file that contains example Prisma Client queries for delegate types
+
+
+## What are Delegate types?
+
+An application such as LinkedIn contains a feed that would contain different types of content such as articles, images, and videos. In such an instance, you would have a *parent entity*, `Feed`, containing all the shared properties and *child entities*,`Article`, `Image`, and `Video`, that inherit the *parent entity*'s properties and have their individual properties. 
```

Comment:
I don't agree that the parent entity would be called `Feed` in this case since the feed itself is a different concept than articles, images or videos — it's basically a _sorted collection_ of these. I think `FeedItem` would be more appropriate.

Reply:
I think this proves naming is indeed difficult 😆 

We could go with `Activity` perhaps. It was used in the [union types issue](https://github.com/prisma/prisma/issues/2505).